tests/test_lib_thread_names.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from datetime import datetime
+
+from juniorguru_chick.lib.threads import name_thread
+
+DAYS = ["Pondělní", "Úterní", "Středeční",
+        "Čtvrteční", "Páteční", "Sobotní", "Nedělní"]
+
+class Message:
+    def __init__(self, display_name, message):
+        self.author = lambda: None
+        self.author.display_name = display_name
+        self.content = message
+
+def test_name_thread_no_brackets():
+    content = "Hello this is my thread"
+    weekday = datetime.now().weekday()
+    name_template = "{weekday} objev od {author}"
+    expected_name = f"{DAYS[weekday]} objev od Jana"
+    message = Message("Jana", content)
+
+    assert name_thread(message, name_template) == expected_name
+
+# test if brackets are not closed and if there is no text in brackets
```

Comment:
Either this comment is redundant or `test_brackets_used_incorrectly` is not clear. If `test_brackets_used_incorrectly` feels clear to you, we can remove this comment. If `test_brackets_used_incorrectly` doesn't feel sufficient, it's completely okay to name tests like `test_if_brackets_are_not_closed_and_if_there_is_no_text_in_brackets` 😄